subsystem/hcp/machine_pool_resource_test.go

```diff
@@ -1867,6 +1885,157 @@ var _ = Describe("Hcp Machine pool", func() {
 			Expect(resource).To(MatchJQ(".attributes.aws_node_pool.instance_type", "r5.xlarge"))
 			Expect(resource).To(MatchJQ(`.attributes.kubelet_configs`, "my_kubelet_config"))
 		})
+
+		It("Can create machine pool with http tokens set and cannot edit marco", func() {
```

Comment:
```suggestion
		It("Can create machine pool with http tokens set and cannot edit", func() {
```

Reply:
oh, thanks for pointing this. I forgot to remove the debug content before commit it